packages/react/navigation-menu/src/NavigationMenu.tsx

```diff
@@ -285,9 +257,11 @@ const NavigationMenuProvider: React.FC<NavigationMenuProviderProps> = (
         });
       }, [])}
     >
-      <ViewportContentProvider scope={scope} items={viewportContent}>
-        {children}
-      </ViewportContentProvider>
+      <Collection.Provider scope={scope}>
```

Comment:
Removed `SubImpl` part as it was only needed to give us access to this collection, since we don't need to access it anymore I moved it back into the common provider.